config.go

```diff
@@ -37,8 +38,9 @@ func newConfigBuilder() *Builder {
 // It panics under the following circumstances:
 //     * target is not a struct pointer
 //     * struct contains unsupported fields (pointers, maps, slice of structs, channels, arrays, funcs, interfaces, complex)
-func (c *Builder) To(target interface{}) {
+func (c *Builder) To(target interface{}) *Builder {
```

Comment:
for brevity, this can just return `error`

Reply:
I thought about that but it seems more consistent to return the builder.

you can check for error at every step of the way by `b.Err()` and it's just consistent to have too be a *Builder for the same.

But that's the only reason I did it that way.